few updates

djdanielsson opened the pull request:
- Removed all references to awx.awx because we no longer support that officially
  - Moved some docs to the docs folder
  - Updated links to use FQDN vs short links so they will work outside of github
fixes #1007
fixes #1085

djdanielsson committed:
modified:   .github/CONTRIBUTING.md
	modified:   .github/collections/requirements.yml
	modified:   .github/workflows/release.yml
	deleted:    CONVERSION_GUIDE.md
	modified:   README.md
	new file:   changelogs/fragments/fix_readmes_requirements.yml
	modified:   docs/CONVERSION_GUIDE.md
	renamed:    EXPORT_README.md -> docs/EXPORT_README.md
	new file:   docs/TOWER_CONVERSION_GUIDE.md
	modified:   roles/ansible_config/README.md
	modified:   roles/controller_ad_hoc_command/README.md
	modified:   roles/controller_ad_hoc_command/meta/main.yml
	modified:   roles/controller_ad_hoc_command/tests/test.yml
	modified:   roles/controller_ad_hoc_command_cancel/README.md
	modified:   roles/controller_ad_hoc_command_cancel/meta/main.yml
	modified:   roles/controller_ad_hoc_command_cancel/tests/test.yml
	modified:   roles/controller_applications/README.md
	modified:   roles/controller_applications/meta/main.yml
	modified:   roles/controller_applications/tests/test.yml
	modified:   roles/controller_bulk_host_create/README.md
	modified:   roles/controller_bulk_host_create/meta/main.yml
	modified:   roles/controller_bulk_job_launch/README.md
	modified:   roles/controller_bulk_job_launch/meta/main.yml
	modified:   roles/controller_credential_input_sources/README.md
	modified:   roles/controller_credential_input_sources/meta/main.yml
	modified:   roles/controller_credential_input_sources/tests/test.yml
	modified:   roles/controller_credential_types/README.md
	modified:   roles/controller_credential_types/meta/main.yml
	modified:   roles/controller_credential_types/tests/test.yml
	modified:   roles/controller_credentials/README.md
	modified:   roles/controller_credentials/meta/main.yml
	modified:   roles/controller_credentials/tests/test.yml
	modified:   roles/controller_execution_environments/README.md
	modified:   roles/controller_execution_environments/meta/main.yml
	modified:   roles/controller_execution_environments/tests/test.yml
	modified:   roles/controller_host_groups/README.md
	modified:   roles/controller_host_groups/meta/main.yml
	modified:   roles/controller_host_groups/tests/test.yml
	modified:   roles/controller_hosts/README.md
	modified:   roles/controller_hosts/meta/main.yml
	modified:   roles/controller_hosts/tests/test.yml
	modified:   roles/controller_instance_groups/README.md
	modified:   roles/controller_instance_groups/meta/main.yml
	modified:   roles/controller_instance_groups/tests/test.yml
	modified:   roles/controller_instances/README.md
	modified:   roles/controller_instances/meta/main.yml
	modified:   roles/controller_instances/tests/test.yml
	modified:   roles/controller_inventories/README.md
	modified:   roles/controller_inventories/meta/main.yml
	modified:   roles/controller_inventories/tests/test.yml
	modified:   roles/controller_inventory_source_update/README.md
	modified:   roles/controller_inventory_source_update/meta/main.yml
	modified:   roles/controller_inventory_source_update/tests/test.yml
	modified:   roles/controller_inventory_sources/README.md
	modified:   roles/controller_inventory_sources/meta/main.yml
	modified:   roles/controller_inventory_sources/tests/test.yml
	modified:   roles/controller_job_launch/README.md
	modified:   roles/controller_job_launch/meta/main.yml
	modified:   roles/controller_job_launch/tests/test.yml
	modified:   roles/controller_job_templates/README.md
	modified:   roles/controller_job_templates/meta/main.yml
	modified:   roles/controller_job_templates/tests/test.yml
	modified:   roles/controller_jobs_cancel/README.md
	modified:   roles/controller_jobs_cancel/meta/main.yml
	modified:   roles/controller_jobs_cancel/tests/test.yml
	modified:   roles/controller_labels/README.md
	modified:   roles/controller_labels/meta/main.yml
	modified:   roles/controller_labels/tests/test.yml
	modified:   roles/controller_license/README.md
	modified:   roles/controller_license/meta/main.yml
	modified:   roles/controller_license/tests/test.yml
	modified:   roles/controller_notification_templates/README.md
	modified:   roles/controller_notification_templates/meta/main.yml
	modified:   roles/controller_notification_templates/tests/test.yml
	modified:   roles/controller_organizations/README.md
	modified:   roles/controller_organizations/meta/main.yml
	modified:   roles/controller_organizations/tests/test.yml
	modified:   roles/controller_project_update/README.md
	modified:   roles/controller_project_update/meta/main.yml
	modified:   roles/controller_project_update/tests/test.yml
	modified:   roles/controller_projects/README.md
	modified:   roles/controller_projects/meta/main.yml
	modified:   roles/controller_projects/tests/test.yml
	modified:   roles/controller_roles/README.md
	modified:   roles/controller_roles/meta/main.yml
	modified:   roles/controller_roles/tests/test.yml
	modified:   roles/controller_schedules/README.md
	modified:   roles/controller_schedules/meta/main.yml
	modified:   roles/controller_schedules/tests/test.yml
	modified:   roles/controller_settings/README.md
	modified:   roles/controller_settings/meta/main.yml
	modified:   roles/controller_settings/tests/test.yml
	modified:   roles/controller_teams/README.md
	modified:   roles/controller_teams/meta/main.yml
	modified:   roles/controller_teams/tests/test.yml
	modified:   roles/controller_users/README.md
	modified:   roles/controller_users/meta/main.yml
	modified:   roles/controller_users/tests/test.yml
	modified:   roles/controller_workflow_job_templates/README.md
	modified:   roles/controller_workflow_job_templates/meta/main.yml
	modified:   roles/controller_workflow_job_templates/tests/test.yaml
	modified:   roles/controller_workflow_launch/README.md
	modified:   roles/controller_workflow_launch/meta/main.yml
	modified:   roles/controller_workflow_launch/tests/test.yml
	modified:   roles/dispatch/tests/test.yml
	modified:   tests/configure_controller.yml
	modified:   tests/configure_controller_export_model.yml
	modified:   tests/templated_role_example/README.md
	modified:   tests/templated_role_example/meta/main.yml
	modified:   tests/templated_role_example/tests/test.yml

djdanielsson commented on .github/workflows/release.yml:
```diff
@@ -27,7 +27,7 @@ jobs:
       collection_repo: https://github.com/redhat-cop/aap_configuration/
       quay_username: redhat_cop
-      collection_dependencies: awx.awx ansible.eda ansible.hub
+      collection_dependencies: ansible.controller ansible.platform ansible.eda ansible.hub
```
not sure if this will break anything

Tompage1994 replied:
@sean-m-sullivan will your new changes on the tooling enable this? Can you point us at what vars will need to be set to achieve it?

sean-m-sullivan replied:
None, I removed it

Tompage1994 commented on CONVERSION_GUIDE.md:
Any reason why we are removing this?

djdanielsson replied:
I'm not, I'm moving it to docs and have 2 different conversations now one for tower_configuration and one for controller_configuration

sean-m-sullivan reviewed (approved):
Lgrm

djdanielsson committed:
modified:   README.md

djdanielsson committed:
Merge branch 'devel' into readme_awx_cleanup

djdanielsson commented:
this should close #1007

djdanielsson committed:
modified:   roles/gateway_authenticator_maps/README.md
	modified:   roles/gateway_authenticators/README.md
	modified:   roles/gateway_organizations/README.md
	modified:   roles/gateway_role_user_assignments/README.md

sean-m-sullivan commented:
Merging this will work on fix post
